src/app/functions/server/onerep.ts

```diff
@@ -2,17 +2,19 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
-import AppConstants from "../appConstants.js";
-import mozlog from "../utils/log.js";
-import { parseE164PhoneNumber, parseIso8601Datetime } from "../utils/parse.js";
-import { usStates } from "../utils/states.js";
+import AppConstants from "../../../appConstants.js";
+import mozlog from "../../../utils/log.js";
+import {
+  parseE164PhoneNumber,
+  parseIso8601Datetime,
+} from "../../../utils/parse.js";
+import { usStates } from "../../../utils/states.js";
 const log = mozlog("external.onerep");
 
-/**
- * @param {string} path
- * @param {Parameters<typeof fetch>[1]} [options]
- */
-async function onerepFetch(path, options = {}) {
+async function onerepFetch(
```

Comment:
nit: should this be `onerepFetch()` or `oneRepFetch()`?

Reply:
If it should be `oneRepFetch` then it should also be `uSStates`, so... I'd stay with `onerepFetch`.